package.json

```diff
@@ -43,15 +43,15 @@
     "package.json": "node .github/dep-version-check"
   },
   "peerDependencies": {
+    "@linusborg/vue-simple-portal": "^0.1.5",
+    "@popperjs/core": "^2.10.2",
+    "@square/maker-icons": "^3.2.0",
     "colord": "^2.9.2",
     "date-fns": "^2.19.0",
     "lodash": "^4.17.20",
     "popmotion": "^9.3.1",
     "stylefire": "^7.0.3",
-    "vue": "2.6.14",
-    "@square/maker-icons": "^3.2.0",
-    "@linusborg/vue-simple-portal": "^0.1.5",
-    "@popperjs/core": "^2.10.2"
+    "vue": ">= 2.6.14 < 3"
```

Comment:
isn't this identical to `"vue": "^2.6.14"`? can we write it that way so it matches the rest of dependency versions in this file?

Reply:
Hmm yeah let me try